src/_routing/_components/RootStackNavigator.tsx

```diff
@@ -3,6 +3,7 @@ import SplashScreen from 'react-native-lottie-splash-screen';
 import { HeaderBackButton } from '@react-navigation/elements';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
+import { CloseWhite } from '../../_assets/icons';
```

Comment:
PP: verwijder de andere close image en gebruik deze als default close, all de andere usages gebruiken toch wss tintColor.. Of gebruik hier een HeaderIcon of HeaderButton met icon component in, ipv een aparte witte png toe te voegen (ik denk zelfs dat er ook een tintIconColor prop (ofzoiets) bestaat voor de header